index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hellow World</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hello! World</h1>
```

Comment:
- [ ] **[OPTIONAL]** Kindly follow the instruction for this exercise 👇🏽 and writhe **"Hello Microverse!"** instead of `Hello! World`

> A .html file that includes a header with the text **"Hello Microverse!"**. Use the h1 tag for that.
![image](https://user-images.githubusercontent.com/58642949/191871581-a87bb6df-28ca-41b8-9b9d-4d6b1c793030.png)